src/Silverback.Integration/Silverback.Integration.csproj

```diff
@@ -34,7 +34,7 @@ This package contains the message broker and connectors abstractions.</Descripti
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.7.1" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="5.0.0" />
```

Comment:
Stupid question but is this gonna work with .net core 3.0 (and even 2.2) apps?

Reply:
I think so (not so sure about 2.2 though).